src/north/CsvToHttp/CsvToHttp.schema.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { notEmpty } from '../../services/validation.service'
+
+const schema = { name: 'CsvToHttp' }
+
+schema.form = {
+  WebParameters: {
+    type: 'OIbTitle',
+    label: 'Http End Point Parameters',
+    children: (
+      <div>
+        <ul>
+          <li>
+            HTTP/HTTPS Url: Url which will be used in order to send the csv file converted into a http request.
+          </li>
+          <li>
+            HTTP/HTTPS Token: Token which permits to connect into the applicative website
+          </li>
+          <li>
+            HTTP authentication: Method choose to authenticate you within the applicative host
+          </li>
+        </ul>
+      </div>
+    ),
+  },
+  applicativeHostUrl: {
+    type: 'OIbLink',
+    protocols: ['http', 'https'],
+    defaultValue: '',
+  },
+  requestMethod: {
+    type: 'OIbSelect',
+    newRow: false,
+    options: ['POST', 'PUT', 'PATCH'],
+    md: 3,
+    defaultValue: 'POST',
+  },
+  authentication: { type: 'OIbAuthentication' },
+  CsvToHttpParameters: {
+    type: 'OIbTitle',
+    label: 'Mapping Parameters',
+    children: (
+      <div>
+        <ul>
+          <li>
+            The unselected index will be obmitted during the mapping
```

Comment:
I think that "index" here is confusing : maybe you mean tag or key or column name ? For me index is the row number